src/pages/ExamsPage/components/AllowanceListActions.jsx

```diff
@@ -10,8 +10,53 @@ import {
 import { DeleteOutline, EditOutline } from '@openedx/paragon/icons';
 import { useIntl } from '@edx/frontend-platform/i18n';
 
+import { Form } from 'react-router-dom';
+import { useState } from 'react';
 import messages from '../messages';
-import { useDeleteAllowance } from '../hooks';
+import { useDeleteAllowance, useEditAllowance } from '../hooks';
+
+// todo: add onEdit
+const EditModal = ({
+  isOpen, onCancel, onEdit,
+}) => {
+  const [ additionalTimeError, setAdditionalTimeError ] = useState(false);
+  const { formatMessage } = useIntl();
+
+  return (
```

Comment:
This part is still very much in progress. Currently trying to figure out why I can't open the modal. I'm wondering if it has something to do with the way the props are passed in? I'm referencing the AddAllowanceModal, but I think I need to set some sort of initial state maybe?

Reply:
A few options to debug. If you just set isOpen to true in the render does it work? That would rule layout/render issues. What value is this component getting passed in for isOpen?